lib/matplotlib/backend_bases.py

```diff
@@ -3059,6 +3059,11 @@ def press_pan(self, event):
 
     def drag_pan(self, event):
         """Callback for dragging in pan/zoom mode."""
+        if self._pan_info.button not in event.buttons:
```

Comment:
Shouldn't this be an exact match? I believe the action should terminate as soon as any button is pressed or released.

Reply:
Sure, I don't really mind either way.